src/python/tools/db2graph/db2graph.py

```diff
@@ -0,0 +1,712 @@
+            print("Error: Incorrect edge entity node - entity node formatting, " +
+                "missing ',' at the end of table1_name.col1_name")
+            exit(1)
+        
+        check_split = qry_split[2].split('.')
+        if (len(check_split) != 2):
+            print("Error: Incorrect edge entity node - entity node formatting, " +
+                "table2_name.col2_name not correctly formatted")
+            exit(1)
+        
+        check_var = qry_split[3].lower()
+        if (check_var != "from"):
+            print("Error: Incorrect edge entity node - entity node formatting, " +
+                "extra elements after table2_name.col2_name")
+            exit(1)
+        
+        new_query_list.append(edge_entity_entity_queries_list[q])
+    
+    return new_query_list
+
+def validation_check_edge_entity_feature_val_queries(edge_entity_feature_val_queries_list):
+    """
+    Ensures that the edge_entity_feature_val_queries_list are correctly formatted.
+
+    :param edge_entity_feature_val_queries_list: List of all the queries defining edges from entity node to feature values
+    :return new_query_list: These are updated queries with necessary changes if any
+    """
+    # Format: SELECT table1_name.col1_name, ____ FROM ____ WHERE ____ (and so on);
+    new_query_list = list()
+    for q in range(len(edge_entity_feature_val_queries_list)):
+        qry_split = edge_entity_feature_val_queries_list[q].split(' ')
+        
+        check_var = qry_split[0].lower()
+        if (check_var != "select"):
+            print("Error: Incorrect edge entity node - feature value formatting, " +
+                "not starting with SELECT")
+            exit(1)
+        
+        check_split = qry_split[1].split('.')
+        if (len(check_split) != 2):
+            print("Error: Incorrect edge entity node - feature value formatting, " +
+                "table1_name.col1_name not correctly formatted")
+            exit(1)
+        if (check_split[1][-1] != ','):
+            print("Error: Incorrect edge entity node - feature value formatting, " +
+                "missing ',' at the end of table1_name.col1_name")
+            exit(1)
+        
+        new_query_list.append(edge_entity_feature_val_queries_list[q])
+    
+    return new_query_list
+
+def clean_token(token):
+    """
+    Helper to clean a dataframe, can be used by applying this function to a dataframe
+
+    :param token: elements to clean
+    :return token: cleaned token
+    """
+    token = str(token)
+    token = token.strip().strip("\t.\'\" ")
+    return token.lower()
+
+def get_uuid(row):
+    """
+    Gets the UUID associated with entity_node on this row consisting of a three-tuple
+
+    :param row: a row in the dataframe
+    :return uuid: the UUID associated with entity_node on this row
+    """
+    return uuid.uuid5(uuid.NAMESPACE_DNS, row['entity_node'])
+
+def entity_node_to_uuids(output_dir, cnx, entity_queries_list, db_server):
```

Comment:
If the user specifically wants to generate UUID from the entity nodes they can specific ``generate_uuid = true`` in the ``config.yaml`` and this code will first generate the uuids for all the entity nodes.
In this fashion the output will not be:
`tableName_columnName_src -> relation -> tableName_columnName_dst`, but will be
`uuid_src -> relation -> uuid_dst` (Here uuid is generated from the function and we also have a mapping output for reference)

Reply:
So I understand what it does, but what use case would require the UUID generation? I'm trying to figure out if this is just functionality for the sake of functionality, or this is something that certain users will need.

Besides making the code more complex, introducing another mapping complications things because in the Marius preprocessing we already do another mapping from entity ids to integer ids in the range [0, num_nodes-1]